packages/grid-export/src/features/grid-copy-to-clipboard/transformers/CellTransformer.ts

```diff
@@ -44,5 +44,5 @@ export const transformItemToCell = <TItem, TItemValue>(
 };
 
 const createCell = (value: string | number): string => {
-  return `<td>${value}</td>`;
+  return `<td style="text-align: left">${value}</td>`;
```

Comment:
Isn't it better to get this from the column config?
justifyContent is available per column.

Reply:
Ok, fixed conversion from justifyContent to align